packages/cna-template/template/frameworks/jest/jest.setup.js

```diff
@@ -0,0 +1,2 @@
+// Enable when you want to import components automatically.
+import "./.nuxt/components/plugin.js";
```

Comment:
I'm not sure this should be enabled by default.

Reply:
Indeed when `components: true` is not provided, this file does not exists... Maybe using `import().catch(()=>{})`? But i think other than this, by default we have `components: true` in default CNA nuxt config